app/main.py

```diff
@@ -16,4 +16,13 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [], "immutable": []}
+sorted_variables["mutable"].append(my_favourite_films)
+sorted_variables["mutable"].append(marks)
+sorted_variables["mutable"].append(collection_of_coins)
```

Comment:
The 'collection_of_coins' set is categorized as mutable, but sets are generally considered immutable in terms of their identity, even though their contents can change. Consider moving 'collection_of_coins' to the 'immutable' category if you are categorizing by identity immutability.